app/infrastracture/customErrors/notFoundError.js

```diff
@@ -0,0 +1,6 @@
+module.exports = class NotFoundError extends Error {
```

Comment:
Also, you can create BadRequestError for 400 status code and check a type of error with instanceof in error middleware